docs/verify_cri.md

```diff
@@ -0,0 +1,100 @@
+# Verify pouch CRI by the cri-tools
+
+[cri-tools](https://github.com/kubernetes-incubator/cri-tools) provides a CLI([crictl](https://github.com/kubernetes-incubator/cri-tools/blob/master/docs/crictl.md)) for CRI-compatible container runtimes. This is an easy way to verify pouch CRI without needing to set up Kubernetes components.
+
+## Install
+
+The CRI CLI can be installed easily via `go get` command:
+
+```bash
+go get github.com/kubernetes-incubator/cri-tools/cmd/crictl
+```
+
+Then `crictl` binary can be found in `$GOPATH/bin`.
+
+*Note: ensure GO is installed and GOPATH is set before installing crictl.*
+
+## Usage
+
+```bash
+crictl SUBCOMMAND [FLAGS]
+```
+
+Subcommands includes:
+
+- `info`:          Display runtime version information
+- `runs`:          Run a new sandbox
+- `stops`:         Stop a running sandbox
+- `rms`:           Remove a sandbox
+- `inspects`:      Display the status of a sandbox
+- `sandboxes`:     List sandboxes
+- `create`:        Create a new container
+- `start`:         Start a created container
+- `stop`:          Stop a running container
+- `rm`:            Remove a container
+- `inspect`:       Display the status of a container
+- `ps`:            List containers
+- `status`:        Display status of the container runtime
+- `attach`:        Attach to a running container
+- `pull`:          Pull an image from a registry
+- `images`:        List images
+- `inspecti`:      Return the status of an image
+- `rmi`:           Remove an image
+- `exec`:          Run a command in a running container
+- `port-forward`:  Forward local port to a sandbox
+- `logs`:          Fetch the logs of a container
+- `help`:          Shows a list of commands or help for one command
+
+## Config runtime endpoint
+
+- By setting flags `--runtime-endpoint` and `--image-endpoint`
+- By setting environment variables `CRI_RUNTIME_ENDPOINT` and `CRI_IMAGE_ENDPOINT`
+- By setting the endpoint in the config file `--config=/etc/crictl.yaml`
+
+```
+# cat /etc/crictl.yaml
+runtime-endpoint: /var/run/pouchcri.sock
+image-endpoint: /var/run/pouchcri.sock
+timeout: 10
+debug: true
+```
+## Examples
+
+### Run sandbox with config file
```

Comment:
As a verification case, we should provide successful case as the `crictl runs sandbox-config.json` command below. While I think we still need to provide some unsuccessful cases which show users what happens if CRI implementation misses something.